From 129a85979f9e624218d19d5f0ba3382ab78ef8a9 Mon Sep 17 00:00:00 2001
From: Martin Liska <mliska@suse.cz>
Date: Fri, 26 May 2023 11:19:05 +0200
Subject: [PATCH 5/6] Stop building against LLVM 11 due to a known LLVM issue:

ld: /usr/lib64/libLLVM.so.11: __morestack: invalid needed version 2
ld: /usr/lib64/libLLVM.so.11: error adding symbols: bad value
---
 .github/workflows/build.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.github/workflows/build.yml b/.github/workflows/build.yml
index 77f5509..a499b66 100644
--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        llvm: [11, 12, 13, 14, 15, 16]
+        llvm: [12, 13, 14, 15, 16]
         build-type: [DEBUG]
         docker: [opensuse/tumbleweed]
         include:
-- 
2.41.0.255.g8b1d071c50-goog

